src/translations/v6/v6.sbvr

```diff
@@ -140,7 +140,7 @@ Term: image location
 	Concept Type: Short Text (Type)
 
 Term: image size
-	Concept Type: Big Integer (Type)
+	Concept Type: Integer (Type)
```

Comment:
Any reason not to leave this as a big integer type here? It shouldn't affect anything other than knowing this is actually a big integer in the db rather than an integer when looking at it
```suggestion
	Concept Type: Big Integer (Type)
```